### PR TITLE
Update 1294585866.rst

### PR DESCRIPTION
**Pull request by franzholz**
Add a hint how to solve this in many cases.

### CODE REVIEW

File: Documentation/Exceptions/1294585866.rst [hunk 1]
```diff
@@ -11,3 +11,6 @@ No class named "..."
 
 Check your source code for occurrences of that class name. Usually happens if a
 hook is registered with a wrong class name/reference.
+
+Solution 1: 
+Try to use the TYPO3 backend Install Tool: Rebuild PHP Autoload Information
```
Comment (sbuerk) [inline Documentation/Exceptions/1294585866.rst:16]: Only suitable/doable for non-composer installations. Composer installations needs run:

```shell
composer dump-autoload
```

or eventually `composer install (--no-dev)` to be safe.

**Review by sbuerk — CHANGES_REQUESTED** [review]

File: Documentation/Exceptions/1294585866.rst (hunk 1 above)
Comment (franzholz) [reply Documentation/Exceptions/1294585866.rst]: I have added now your proposal for composer.

File: Documentation/Exceptions/1294585866.rst [hunk 2]
```diff
@@ -11,3 +11,13 @@ No class named "..."
 
 Check your source code for occurrences of that class name. Usually happens if a
 hook is registered with a wrong class name/reference.
+
+Solution 1:
```
Comment (brotkrueml) [inline Documentation/Exceptions/1294585866.rst:15]:
```suggestion
Possible solution:
```

As there is currently only one solution. If there is a 1 I would also expect a 2.

File: Documentation/Exceptions/1294585866.rst [hunk 3]
```diff
@@ -11,3 +11,13 @@ No class named "..."
 
 Check your source code for occurrences of that class name. Usually happens if a
 hook is registered with a wrong class name/reference.
+
+Solution 1: 
+non Composer:
+
+*   Try to use the TYPO3 backend Install Tool: Rebuild PHP Autoload Information
```
Comment (brotkrueml) [inline Documentation/Exceptions/1294585866.rst:18]:
```suggestion
*   Try to use the :guilabel:`Admin Tools > Maintenance` backend module: Rebuild PHP Autoload Information
```

File: Documentation/Exceptions/1294585866.rst [hunk 4]
```diff
@@ -11,3 +11,13 @@ No class named "..."
 
 Check your source code for occurrences of that class name. Usually happens if a
 hook is registered with a wrong class name/reference.
+
+Solution 1: 
+non Composer:
```
Comment (brotkrueml) [inline Documentation/Exceptions/1294585866.rst:16]:
```suggestion
Legacy installation:
```

File: Documentation/Exceptions/1294585866.rst [hunk 5]
```diff
@@ -11,3 +11,13 @@ No class named "..."
...
+
+*   Try to use the TYPO3 backend Install Tool: Rebuild PHP Autoload Information
+
+Composer:
```
Comment (brotkrueml) [inline Documentation/Exceptions/1294585866.rst:20]:
```suggestion
Composer-based installation:
```

**Review by brotkrueml — CHANGES_REQUESTED** [review]

**Review by brotkrueml — APPROVED** [review]

**Review by sbuerk — APPROVED** [review]

**Comment by sbuerk** [comment]
@brotkrueml you requested review after merging ?

**Comment by brotkrueml** [comment]
> @brotkrueml you requested review after merging ?

@sbuerk Sorry, could not merge after my approval because of your approval was missing. But in that minute I checked what was wrong, you approved. So this was just the same time. Sorry.

**Pull request merged**